protostar/protostar_cli.py

```diff
@@ -15,6 +17,13 @@
 from protostar.utils.log_color_provider import LogColorProvider
 
 
+def _consume_pythonpath():
+    pythonpath_env_var = os.environ.get("PYTHONPATH") or ""
+    split_paths = pythonpath_env_var.split(":")
+    if split_paths != [""]:
+        sys.path.extend(split_paths)
```

Comment:
Edge cases I don't know if are dealt with properly:
1. I'm not sure how this will work with `~` in the path (maybe https://docs.python.org/3/library/os.path.html#os.path.expanduser must also be called?)
2. How to add a path which contains `:` within?

Reply:
1. Tested - doesn't need to be called
2. I think this would be rare enough to just ignore it for the time being. If somebody reports this as an issue we should circle back on this

Reply:
👍🏻